applications/StructuralMechanicsApplication/custom_utilities/structural_mechanics_element_utilities.cpp

```diff
@@ -8,20 +8,166 @@
 //					 Kratos default license: kratos/license.txt
 //
 //  Main authors:    Philipp Bucher
+//                   Vicente Mataix Ferrandiz
+//                   Riccardo Rossi
+//                   Ruben Zorrilla
 //
 
 // System includes
 
 // External includes
 
 // Project includes
+#include "includes/checks.h"
 #include "structural_mechanics_element_utilities.h"
 #include "structural_mechanics_application_variables.h"
 #include "utilities/math_utils.h"
 
 namespace Kratos {
 namespace StructuralMechanicsElementUtilities {
 
+int BaseElementCheck(
```

Comment:
I found that `BaseElementCheck` is a misleading name as this is not the base class of the elements that use it. Something like `SolidElementCheck`, `BaseCheck` o `StandardCheck` sounds better to me.

Reply:
OK